### PR TITLE
Paper

### PR DESCRIPTION
**Pull request by LouiseDck**
- Adds manuscript
- Fixes citation on website
- Adds compiled paper (using [openjournals/inara](https://github.com/openjournals/inara))

### CODE REVIEW

File: paper/utils.Rmd [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: Utils for the paper
```
Comment (rcannood) [inline paper/utils.Rmd:2]: Why have a separate utils.Rmd vs. just including this code in the paper.Rmd? :thinking:

**Review by rcannood — CHANGES_REQUESTED** [review]

File: inst/CITATION [hunk 2]
```diff
@@ -33,10 +33,50 @@ bibentry(
...
+      role = c("aut"),
+      comment = c(ORCID = "0000-0001-7858-6521", github = "artuurC")
+    ),
+    person(
+      "Nikolay",
+      "S. Markov",
+      email = "nikolai.markov@icloud.com",
+      role = c("aut"),
+      comment = c(ORCID = "0000-0002-3659-4387", github = "mxposed")
+    ),
```
[the remark this answers is not in the document]
Comment (LouiseDck) [reply inst/CITATION:60]: Good point!

File: paper/utils.Rmd (hunk 1 above)
Comment (LouiseDck) [reply paper/utils.Rmd:2]: The paper.md is not valid .Rmd (there were issues with the headers), so I decided to separate out the generation of the figures and tables.
I can rename this file to paper.Rmd, but I think this  is clear too.

File: paper/utils_files/figure-gfm/fig-dynbenchmark-1.png
Comment (rcannood) [inline paper/utils_files/figure-gfm/fig-dynbenchmark-1.png:1]: this png can now be removed! gj on switching to svg :+1:

File: paper/utils.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    ## ✖ dplyr::lag()    masks stats::lag()
+    ## ℹ Use the conflicted package (<http://conflicted.r-lib.org/>) to force all conflicts to become errors
+
+``` r
+legends <- list(
+  list(title = "Bar", palette = "qc", enabled = FALSE, geom = "bar"),
+  list(title = "Bar", palette = "benchmark", enabled = FALSE, geom = "bar"),
+  list(title = "Bar", palette = "scaling", enabled = FALSE, geom = "bar"),
+  list(title = "Bar", palette = "stability", enabled = FALSE, geom = "bar"),
+  list(title = "Error reason", palette = "error_reasons", enabled = TRUE, geom = "pie"),
```
Comment (rcannood) [inline paper/utils.md:23]: I forget -- do we have knobs to make this geom wider?

File: paper/utils.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    ## ✖ dplyr::filter() masks stats::filter()
+    ## ✖ dplyr::lag()    masks stats::lag()
+    ## ℹ Use the conflicted package (<http://conflicted.r-lib.org/>) to force all conflicts to become errors
+
+``` r
+legends <- list(
+  list(title = "Bar", palette = "qc", enabled = FALSE, geom = "bar"),
+  list(title = "Bar", palette = "benchmark", enabled = FALSE, geom = "bar"),
+  list(title = "Bar", palette = "scaling", enabled = FALSE, geom = "bar"),
+  list(title = "Bar", palette = "stability", enabled = FALSE, geom = "bar"),
```
Comment (rcannood) [inline paper/utils.md:22]: I don't see these geoms in the svg; do we need to rerender?

File: paper/paper.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+bibliography: paper.bib
+---
+
+*: Shared first authors, †: Shared last authors.
+
+# Summary
+
+The `{funkyheatmap}` package offers a flexible and user-friendly solution for visualising data frames containing a mixture of categorical, proportional, and text-based data. It simplifies the creation of informative and visually appealing heatmaps while providing extensive customization options to tailor the output. This tool is especially valuable in research settings for summarising and communicating complex results, such as those encountered in benchmarking studies.
+
+The package is available on [CRAN](https://cran.r-project.org/package=funkyheatmap) and [PyPI](https://pypi.org/project/funkyheatmappy/) and has a JavaScript port in development. For detailed examples and vignettes, visit the project website [funkyheatmap.github.io](https://funkyheatmap.github.io).
```
Comment (rcannood) [inline paper/paper.md:113]: I guess that technically speaking the project website no longer offers detailed examples and vignettes but a showcase of examples and an overview of the different implementations. WDYT?

```suggestion
The package is available on [CRAN](https://cran.r-project.org/package=funkyheatmap) and [PyPI](https://pypi.org/project/funkyheatmappy/) and has a JavaScript port in development. For a showcase of examples and an overview of the different implementations, visit the project website [funkyheatmap.github.io](https://funkyheatmap.github.io).
```

File: paper/utils.md (hunk 4 above)
Comment (LouiseDck) [reply paper/utils.md:22]: Nope, as you can see they're explicitly not enabled. If this is not added, `verify_legends` detects that there are palettes without legends and will plot these bar geoms.

File: paper/utils.md (hunk 3 above)
Comment (LouiseDck) [reply paper/utils.md:23]: Not for the bar geom legend, and I presume not for any legend? It could be an interesting addition.

Comment (rcannood) [reply paper/utils.md:23]: Sorry, I didn't mean to highlight the bar geoms.

What I noticed is that the pie legend is cut off:

Screenshot from 2024-12-13 16-27-24

File: paper/utils.md (hunk 4 above)
Comment (rcannood) [reply paper/utils.md:22]: I see, good point. Would we want to add a legend for the different parts? The geoms were not added before mainly due to them not being available yet.

File: paper/utils.md (hunk 3 above)
Comment (LouiseDck) [reply paper/utils.md]: Ah right, yeah we do have a setting for that ^^

File: paper/utils.md (hunk 4 above)
Comment (LouiseDck) [reply paper/utils.md:22]: I explicitly didn't want to, to recreate the relevant parts of this figure: https://www.nature.com/articles/s41587-019-0071-9/figures/8

**Review by rcannood — APPROVED** [review]
LGTM!

**Pull request merged**